listeners.go

```diff
@@ -472,16 +473,17 @@ func (na NetworkAddress) ListenQUIC(ctx context.Context, portOffset uint, config
 		// http3.ConfigureTLSConfig only uses this field and tls App sets this field as well
 		//nolint:gosec
 		quicTlsConfig := &tls.Config{GetConfigForClient: sqs.getConfigForClient}
-		earlyLn, err := quic.ListenEarly(h3ln, http3.ConfigureTLSConfig(quicTlsConfig), &quic.Config{
-			Allow0RTT: true,
-			RequireAddressValidation: func(clientAddr net.Addr) bool {
-				// TODO: make tunable?
-				return sqs.getActiveRequests() > 1000
-			},
-		})
+		// Require clients to verify their source address when we're handling more than 256 handshakes per second.
+		limiter := rate.NewLimiter(256, 256)
```

Comment:
Not sure if this value makes sense, I just made up a number.

Reply:
I'll increase the number to 1000. Just like originally done by matt.

Reply:
I don't know the right number either :smile: 1000/sec seems like a lot. But we can try it!